core/modules/layout/layout.admin.inc

```diff
@@ -2999,9 +2999,28 @@ function layout_block_list() {
  */
 function layout_get_block_usage($module = NULL, $delta = NULL) {
   $blocks = &backdrop_static(__FUNCTION__, array());
+
+  $all_template_info = layout_get_layout_template_info(NULL, TRUE);
+
   if (empty($blocks)) {
     $layout_info = layout_load_all();
     foreach ($layout_info as $layout) {
+
+      $layout_template = $layout->layout_template;
+      $layout_has_flexible_template = !empty($all_template_info[$layout_template]['flexible']);
+
+      // Layouts that use flexible templates return cryptic UUIDs as region
+      // names. Replace these UUIDs with the respective user-friendly labels.
+      if ($layout_has_flexible_template) {
+        $flexi_regions = array_flip($all_template_info[$layout_template]['regions']);
+
+        foreach ($flexi_regions as $flexi_region_name => $uuid) {
+          $flexi_regions[$flexi_region_name] = $layout->positions[$uuid];
+        }
+
+        $layout->positions = $flexi_regions;
```

Comment:
Let's use a different variable instead of overridding `$layout->positions`. There's a chance that if this layout were saved back to config with `$layout->save()`, it would corrupt the config data.

Reply:
👍 ...I have fixed this in https://github.com/backdrop/backdrop/pull/2925